pkg/controller/populators/import-populator.go

```diff
@@ -160,7 +160,10 @@ func (r *ImportPopulatorReconciler) reconcileTargetPVC(pvc, pvcPrime *corev1.Per
 		}
 
 		if cc.IsPVCComplete(pvcPrime) && cc.IsUnbound(pvc) {
-			// Once the import is succeeded, we copy labels and rebind the PV from PVC to target PVC
+			// Once the import is succeeded, we copy annotations and labels and rebind the PV from PVC to target PVC
```

Comment:
We will update the annotations again after the rebind, are we sure we need that? Having three `updatePVCWithPVCPrimeAnnotations` calls in the same function makes this harder to maintain. It's not this PR's fault though. Maybe we could move the two updateAnnotations and updateLabels just above the switch?

Reply:
Nevermind, `UpdatesMultistageImportSucceeded` updates the PVC prime annotations and we would need to call it afterwards

Reply:
What about having one at the top before entering the switch and another one after checking `UpdatesMultistageImportSucceeded`? That would leave us with two calls. Not a must though, I'll lgtm and we can consider improving this code a bit in the near future.